pkg/deploy/deploy.go

```diff
@@ -488,4 +388,132 @@ func OperatorExists(cli client.Client, operatorPrefix string) (bool, error) {
 	return false, nil
 }
 
+func getResource(ctx context.Context, cli client.Client, obj *unstructured.Unstructured) (*unstructured.Unstructured, error) {
+	found := obj.DeepCopy()
+	err := cli.Get(ctx, types.NamespacedName{Name: obj.GetName(), Namespace: obj.GetNamespace()}, found)
+	return found, err
+}
+
+func handleDisabledComponent(ctx context.Context, cli client.Client, found *unstructured.Unstructured, componentName string, owner metav1.Object) error {
+	if found == nil {
```

Comment:
Probably, will not work here since line 392 makes allocation.

Reply:
It should since we pass it function argument

Reply:
But it never nil even if the resource does not exist